data/logs_model/splunk_logs_model.py

```diff
@@ -46,7 +46,7 @@ def log_action(
             return
 
         if repository_name is not None:
-            if repository is None or namespace_name is not None:
+            if repository is not None or namespace_name is None:
```

Comment:
is this a bugfix? Just curious 🤔

Reply:
I had incorrect conditional before. I'm following what elastic search does [here](https://github.com/quay/quay/blob/74d6d82748df7fc9b5dcb5f1e712fd3be1b7b38f/data/logs_model/document_logs_model.py#L479-L480)